### ISSUE TITLE
Documentation within the book regarding default modules.

### ISSUE THREAD

**Issue by lupan** [rune-rs/rune#262]
The title is clear, I simply feel like it would be highly advantageous to have a section or even its own category regarding the default modules within the language, for example I was surprised to see std::bytes which I do admit is very useful, but considering the only reference I saw to it was within the example scripts; I was surprised to see it working perfectly well within the playground etc.
side note, the field functions section does not include "copy"

**Comment by udoprog** [comment]
This is correct. It's unclear how these should be documented without something like rustdoc for rune. But it is needed!

**Comment by SWW13** [comment]
I'm using [fundoc](https://github.com/CSSSR/fundoc) for my own API documentation. It has some quirks but allows adding the documentation within the source code. Probably a good location to start documenting and later move on the something more `rustdoc`-like.
